doc/nrfx/nrfx.doxyfile.in

```diff
@@ -1100,7 +1101,7 @@ IGNORE_PREFIX          =
 # If the GENERATE_HTML tag is set to YES, doxygen will generate HTML output
 # The default value is: YES.
 
-GENERATE_HTML          = NO
+GENERATE_HTML          = YES
```

Comment:
does this really make some warnings disappear? is this a known doxygen bug?

Reply:
I have been looking for diff between Doxyfile from NCS and the one we have in nrfx. We have this symbol set to `YES` in nrfx that we don't observe such issues in there, so I have tried that in NCS context and it helped. I am not sure whether this is related to any specific bug

Reply:
@gmarull I have found someone stumbled upon similar issue here: https://stackoverflow.com/questions/37900027/doxygen-parameters-of-member-my-function-are-not-all-documented unfortunately without any specific conclusion

Reply:
Doxygen mysteries, let's go with it for now.